Library/Formula/c.rb

```diff
@@ -0,0 +1,16 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.06.tar.gz"
+  sha1 "0eefb3523abccea7ebdd2c56d4d7be6f23585645"
+
+  depends_on "gnu-sed"
```

Comment:
Ok, can you link that in the formula as a comment? Also, it looks like if we applied that commit as a patch (or waited for a new release containing it) we could get rid of this dependency?